vendor/bat-native-ads/src/bat/ads/internal/account/issuers/issuers_util.cc

```diff
@@ -108,16 +124,18 @@ absl::optional<IssuerInfo> GetIssuerForType(const IssuersInfo& issuers,
 
 bool PublicKeyExistsForIssuerType(const IssuerType issuer_type,
                                   const std::string& public_key) {
-  const IssuersInfo& issuers = GetIssuers();
+  const absl::optional<IssuersInfo> issuers = GetIssuers();
+  if (!issuers) {
+    return false;
+  }
 
-  const absl::optional<IssuerInfo>& issuer_optional =
-      GetIssuerForType(issuers, issuer_type);
-  if (!issuer_optional) {
+  const absl::optional<IssuerInfo>& issuer =
```

Comment:
nit: reference is not needed

Reply:
Fixed. Also fixed other refs for issuers